content/en/blog/2024/elastic-contributes-continuous-profiling-agent.md

```diff
@@ -0,0 +1,152 @@
+whole system, always-on solution that observes code and third-party libraries,
+kernel operations, and other code you don't own. It eliminates the need for code
+instrumentation (run-time/bytecode), recompilation, or service restarts with low
+overhead, low CPU (~1%), and memory usage in production environments.
+
+The Elastic profiling agent facilitates identifying non-optimal code paths,
+uncovering "unknown unknowns", and provides comprehensive visibility into the
+runtime behavior of all applications. Elastic’s continuous profiling agent
+provides support for a wide range of runtimes and languages, such as:
+
+- C/C++
+- Rust
+- Zig
+- Go
+- Java
+- Python
+- Ruby
+- PHP
+- Node.js
+- V8
+- Perl
+- .NET
+
+Additionally, organizations can meet sustainability objectives by minimizing
+computational wastage, ensuring seamless alignment with their strategic ESG
+goals.
+
+## Benefits to OpenTelemetry
+
+This contribution not only boosts the standardization of continuous profiling
+for observability but also accelerates its adoption as a key signal in
+OpenTelemetry. Customers benefit from a vendor-agnostic method of collecting
+profiling data correlating it with existing signals, like tracing, metrics, and
+logs, opening new potential for observability insights and a more efficient
+troubleshooting experience.
+
+### User Benefits of OpenTelemetry Profiling
+
+OpenTelemetry-based continuous profiling unlocks the following possibilities for
+users:
+
+- Improved customer experience: delivering consistent service quality and
+  performance through continuous profiling ensures customers have an application
+  that performs optimally, remains responsive, and is reliable.
+
+- Maximize gross margins: Businesses can optimize their cloud spend and improve
+  profitability by reducing the computational resources needed to run
+  applications. Whole system continuous profiling identifies the most expensive
+  functions (down to the lines of code) across diverse environments that may
+  span multiple cloud providers. In the cloud context, every CPU cycle saved
+  translates to money saved.
+
+- Minimize environmental impact: energy consumption associated with computing is
+  a growing concern (source:
+  [MIT Energy Initiative](https://energy.mit.edu/news/energy-efficient-computing/)).
+  More efficient code translates to lower energy consumption, contributing to a
+  reduction in carbon (CO2) footprint.
+
+- Accelerate engineering workflows: continuous profiling provides detailed
+  insights to help troubleshoot complex issues faster, guide development, and
+  improve overall code quality.
+
+- Improved vendor neutrality: a vendor-agnostic eBPF-based profiling agent
+  removes the need to use proprietary APM agents to collect profiling telemetry.
+
+With these benefits, SREs, devs, and DevOps, can now manage the overall
+application’s efficiency on the cloud while ensuring their engineering teams
+optimize it.
```

Comment:
This speaks to a buyer persona / management, but our audience are technical, so I would rather prefer a different set of advantages. I think it would be especially important to have make clear how profiling complements the other signals

Reply:
Sure, I would suggest we use these:

- Improved customer experience: delivering consistent service quality and
  performance through continuous profiling ensures customers have an application
  that performs optimally, remains responsive, and is reliable.

- Accelerate engineering workflows: continuous profiling provides detailed
  insights to help troubleshoot complex issues faster, guide development, and
  improve overall code quality.

- Improved vendor neutrality: a vendor-agnostic eBPF-based profiling agent
  removes the need to use proprietary APM agents to collect profiling telemetry.

As an SRE these are the simplest advantages that make sense out of that list

Reply:
Let me know what you think

Reply:
@svrnm revised this section with a set of more technical benefits

Reply:
Thanks!